src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -101,17 +81,17 @@ protected void ReadOnly_FileSystemHelper(Action<string> testAction, string subDi
             string readOnlyDirectory = GetTestFilePath();
             Directory.CreateDirectory(readOnlyDirectory);
 
-            Assert.Equal(0, RunAsSudo($"mount --bind {sourceDirectory} {readOnlyDirectory}"));
+            Assert.Equal(0, AdminHelpers.RunAsSudo($"mount --bind {sourceDirectory} {readOnlyDirectory}"));
```

Comment:
Maybe RunAsSudo itself should handle asserting the return value. (If you later needed a way to assert a different value, that could be another parameter)